packages/application-shell/src/components/fetch-project/fetch-project.graphql

```diff
@@ -27,6 +27,15 @@ query ProjectQuery($projectKey: String!) {
       name
       value
     }
+    allAppliedDataFences {
+      ... on StoreDataFence {
+        type
+        name
+        value
```

Comment:
I updated in the MC-BE to call this `value` and not `storeKey`. This is avoid domain specific checks when we map them to applicable dataFences that we can read from in `injectAuthorized`